depthai_helpers/arg_manager.py

```diff
@@ -123,6 +123,7 @@ def parseArgs():
                                                                                                                  "If set to \"high\", the output streams will stay uncompressed\n"
                                                                                                                  "If set to \"low\", the output streams will be MJPEG-encoded\n"
                                                                                                                  "If set to \"auto\" (default), the optimal bandwidth will be selected based on your connection type and speed")
+    parser.add_argument('-guit', '--guiType', type=str, default="auto", choices=["auto", "qt", "opencv"], help="Specify GUI type of the demo. \"opencv\" uses buil-in OpenCV display methods, \"qt\" uses Qt to display interactive GUI. \"auto\" will use OpenCV for Raspberry Pi and Qt for other platforms")
```

Comment:
Typo: `buil-in`

We'd probably have to recommend `"-guit opencv"` to users for some cases (e.g. at the moment simultaneous view of multiple streams). No strong preference, but I'd change the option `opencv` to `cv` just to be easier to type. And maybe also `-guit` to `-gui` or `-gt`, but partial matching for it works already (e.g. `-gui` or `-g`).